dso/model.py

```diff
@@ -355,6 +357,12 @@ def key(self) -> str:
         return self.finding.key
 
 
+@dataclasses.dataclass(frozen=True)
+class SastFinding(Finding):
+    sast_status: str
+    component_context: str
```

Comment:
I think we should use enums for both the `sast_status` and the `component_context`, as both only allow a set of specific values, don't they? 🤔